Content.Server/Strip/StrippableComponent.cs

```diff
@@ -47,8 +47,8 @@ protected override void Initialize()
 
             _strippableSystem = EntitySystem.Get<StrippableSystem>();
             Owner.EnsureComponentWarn<ServerInventoryComponent>();
-            var cuffed = Owner.EnsureComponentWarn<CuffableComponent>();
-            cuffed.OnCuffedStateChanged += UpdateState;
```

Comment:
uhhh, i dunno what this change has to do with the pr and i fear this might break a few things

Reply:
It's to make it so drones don't have to have the ability to be cuffed just to have their hat be removable.

Reply:
this shouldnt break anything, everything else that has strippable already has cuffable, this just makes it so it's actually optional

Reply:
sgtm